src/main/java/com/minecolonies/coremod/entity/pathfinding/PathNavigate.java

```diff
@@ -3,7 +3,8 @@
 import com.minecolonies.api.util.BlockPosUtil;
 import com.minecolonies.coremod.entity.EntityCitizen;
 import com.minecolonies.api.util.BlockUtils;
-import com.minecolonies.coremod.util.Log;
+import com.minecolonies.api.util.Log;
+import com.minecolonies.coremod.entity.EntityCitizen;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this duplicated import. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)